src/SignalR/common/Shared/SystemTextJsonExtensions.cs

```diff
@@ -24,10 +25,15 @@ public static void EnsureObjectStart(this ref Utf8JsonReader reader)
         {
             if (reader.TokenType != JsonTokenType.StartObject)
             {
-                throw new InvalidDataException($"Unexpected JSON Token Type '{GetTokenString(reader.TokenType)}'. Expected a JSON Object.");
+                throw new InvalidDataException($"Unexpected JSON Token Type '{reader.GetTokenString()}'. Expected a JSON Object.");
             }
         }
 
+        public static string GetTokenString(this ref Utf8JsonReader reader)
```

Comment:
Who calls this? Do we need to keep the JsonTokenType overload public?

Reply:
Yes, it's used at least once